SwipePopViewController/Classes/SwipePopViewController.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  SwipePopViewController.swift
+//
+//  Created by Athipat Nampetch on 22/1/2564 BE.
+//
+
+import UIKit
+import Foundation
+
+fileprivate func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
+    switch (lhs, rhs) {
+    case let (l?, r?):
+        return l < r
+    case (nil, _?):
+        return true
+    default:
+        return false
+    }
+}
+
+fileprivate func > <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
+    switch (lhs, rhs) {
+    case let (l?, r?):
+        return l > r
+    default:
+        return rhs < lhs
+    }
+}
+
+enum ViewAssociatedKeys {
+    static var firstTranslation = "first_translation"
+    static var panGestureRecognizer = "pan_gesture_recognizer"
+    static var selfNavigationControllerDelegate = "self_navigation_controller_delegate"
+    static var percentDrivenInteractiveTransition = "percent_drivenInteractive_transition"
+    static var panGestureInteractor = "pan_gesture_interactor"
```

Comment:
Add prefix "swipepop." to all string values here to prevent name crash with other libraries